src/esp/scene/ReplicaSemanticScene.cpp

```diff
@@ -112,8 +112,8 @@ bool SemanticScene::buildReplicaHouse(const io::JsonDocument& jsonDoc,
         Sophus::SE3f{worldRotation, vec3f::Zero()} *
         Sophus::SE3f{rotationBoxToWorld, translationBoxToWorld};
 
-    object->obb_ = geo::OBB{transformBoxToWorld * aabbCenter, aabbSizes,
-                            transformBoxToWorld.so3().unit_quaternion()};
+    object->setObb(transformBoxToWorld * aabbCenter, aabbSizes,
```

Comment:
We may want to generalize this by adding an optional transform param to the OBB(AABB) constructor.